gui/default/syncthing/core/syncthingController.js

```diff
@@ -2117,7 +2117,7 @@ angular.module('syncthing.core')
                 folderCfg.versioning.params.command = '' + folderCfg._guiVersioning.externalCommand;
                 break;
             default:
-                delete folderCfg.versioning;
+                folderCfg.versioning.type = '';
```

Comment:
This means we retain various options like staggered times and whatnot but with empty time? Or is that cleaned up in some other place? (I guess `folderCfg.versioning = {type: ''} // nothing` would resolve this otherwise.)